src/zhinst/toolkit/driver/devices/pqsc.py

```diff
@@ -196,42 +208,40 @@ def _check_zsync_connection(
 
         Args:
             ports: Port number to check the ZSync connection for.
-            timeout: Maximum time in seconds the program waits (default: 30).
-            sleep_time: Time in seconds to wait between requesting the status
-                (default: 1)
+            timeout: Maximum time in seconds the program waits.
+            sleep_time: Time in seconds to wait between requesting the status.
 
         Raises:
             TimeoutError: If the process of establishing a ZSync connection the
                 specified port exceeds the specified timeout.
```

Comment:
this changes the interface of the function so we should add a 

.. versionchanged:: 0.6.1

Reply:
done.